js/modulestest/testmodules/events/events.go

```diff
@@ -0,0 +1,157 @@
+// Package events is implementing setInterval setTimeout and co. Not to be used mostly for testing purposes
+package events
```

Comment:
Minor nitpick, but does this package even need to be under `/js`? Why not under `/cmd/integration_tests/eventloop/`?

```suggestion
// Package events implements setInterval, setTimeout and co. Not to be used, mostly for testing purposes
package events
```

Reply:
The original idea was that I will somehow keep the whole `js` integration tests in ... `js` but this means that under `js` I need to import *at least* `./core/local` (or what it is currently there). Which just doesn't seemed like a good idea - so I moved the tests outside of `js` (or didn't move them in it more accurately) but the test extension/module was left there. 

And now currently this package does test itself with a unit tests, using the changes in this PR to make it easier to test evetnt loop related functionaility in the module itself. So it still seems like a good idea to keep it there. But I can move it I guess :shrug:

Reply:
It's not a big deal, but if don't actually plan to expose this to scripts, it seems better to keep it next to the test that actually uses it. It's own unit tests will swill work there, and nothing uses it here, so it feels tidier :man_shrugging: